pkg/queue/instance_test.go

```diff
@@ -108,33 +107,6 @@ func TestRetry(t *testing.T) {
 	wg.Wait()
 }
 
-func TestRetryWithBackoff(t *testing.T) {
```

Comment:
Can we move this test to exponential_test and add a test case for RetryWithContext as that is the replacement for this?

Reply:
seems ok